roles/cockpit-ui/tasks/main.yml

```diff
@@ -1,29 +1,7 @@
 ---
-- name: Create temp directory for kubeconfig
-  command: mktemp -d /tmp/openshift-ansible-XXXXXX
-  register: mktemp
-  changed_when: False
-
-- set_fact:
-    openshift_hosted_kubeconfig: "{{ mktemp.stdout }}/admin.kubeconfig"
-
-- name: Copy the admin client config(s)
-  command: >
-    cp {{ openshift_master_config_dir }}/admin.kubeconfig {{ openshift_hosted_kubeconfig }}
-  changed_when: False
-
-- name: Determine if docker-registry service exists
-  command: >
-    {{ openshift.common.client_binary }} get svc/docker-registry
-    --config={{ openshift_hosted_kubeconfig }}
-    -n default
-  register: check_docker_registry_exists
-  failed_when: false
-  changed_when: false
-
 - name: Create passthrough route for docker-registry
   oc_route:
-    kubeconfig: "{{ openshift_hosted_kubeconfig }}"
+    kubeconfig: "{{ openshift_master_config_dir }}/admin.kubeconfig }}"
```

Comment:
Add `register: docker_registry_route` here, unless I've read the module wrong, it should return back the same data that you would get with a separate list.